pkg/distro/rhel/rhel10/distro_test.go

```diff
@@ -405,7 +405,7 @@ func TestRhel10_KernelOption_NoIfnames(t *testing.T) {
 			for _, imgTypeName := range arch.ListImageTypes() {
 				imgType, err := arch.GetImageType(imgTypeName)
 				assert.NoError(t, err)
-				assert.NotContains(t, imgType.(*rhel.ImageType).KernelOptions, "net.ifnames=0", "type %s contains unwanted net.ifnames=0", imgType.Name())
+				assert.NotContains(t, imgType.(*rhel.ImageType).DefaultImageConfig.KernelOptions, "net.ifnames=0", "type %s contains unwanted net.ifnames=0", imgType.Name())
```

Comment:
I think that the issue here is that `DefaultImageConfig` does not need to be set. You should probably call `imgType.(*rhel.ImageType).getDefaultImageConfig()`. However, note that this file uses a different package name, so calling the unexported method won't work.